src/Text/Pandoc/Writers.hs

```diff
@@ -67,6 +67,7 @@ module Text.Pandoc.Writers
     , writeTEI
     , writeTexinfo
     , writeTextile
+	, writeTxt2Tags
```

Comment:
This should line up with the other entries

Reply:
yes, that's strange because I got some compilation warnings into other places, which I fixed, and I believed it was clean when I've submitted the PR. I'll check this soon.